UnityProject/Assets/Scripts/Variable Viewer/VariableViewerNetworking.cs

```diff
@@ -205,7 +205,7 @@ public int GetSize()
 			return sizeof(ulong)
 				+ sizeof(char) * Title.Length
 				+ sizeof(char) * BookClassname.Length
-				+ BindedPages.Sum(x => x.GetSize())
+				+ (BindedPages == null ? 0 : BindedPages.Sum(x => x.GetSize()))
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
VariableViewerNetworking+NetFriendlySentence.GetSize () (at /github/workspace/UnityProject/Assets/Scripts/Variable Viewer/VariableViewerNetworking.cs:101)
VariableViewerNetworking+NetFriendlyPage+<>c.<GetSize>b__7_0 (VariableViewerNetworking+NetFriendlySentence x) (at /github/workspace/UnityProject/Assets/Scripts/Variable Viewer/VariableViewerNetworking.cs:174)
System.Linq.Enumerable.Sum[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] selector) (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
VariableViewerNetworking+NetFriendlyPage.GetSize () (at /github/workspace/UnityProject/Assets/Scripts/Variable Viewer/VariableViewerNetworking.cs:170)
VariableViewerNetworking.ProcessBook (Librarian+Book _book) (at /github/workspace/UnityProject/Assets/Scripts/Variable Viewer/VariableViewerNetworking.cs:332)
BookNetMessage.Send (Librarian+Book _book, UnityEngine.GameObject ToWho) (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/VariableViewer/BookNetMessage.cs:22)
VariableViewer.SendBookToClient (Librarian+Book Book, UnityEngine.GameObject ToWho) (at /github/workspace/UnityProject/Assets/Scripts/Variable Viewer/BookViewer/VariableViewer.cs:117)
VariableViewer.RequestSendBook (System.UInt64 BookID, UnityEngine.GameObject ByWho) (at /github/workspace/UnityProject/Assets/Scripts/Variable Viewer/BookViewer/VariableViewer.cs:216)
OpenBookIDNetMessage.ValidateAdmin () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Client/VariableViewer/OpenBookIDNetMessage.cs:21)
OpenBookIDNetMessage.Process () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Client/VariableViewer/OpenBookIDNetMessage.cs:14)
GameMessageBase.Process (Mirror.NetworkConnection sentBy) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
ClientMessage.Process (Mirror.NetworkConnection sentBy) (at /